src/components/Menu/Menu.style.ts

```diff
@@ -139,7 +139,7 @@ export const MenuPaper = styled(Paper, {
     maxHeight: 'calc( 100vh - 72px - 12px )',
     borderRadius: '12px !important',
     width: isWide ? '320px' : '288px',
-    marginTop: '-2px',
+    marginTop: `${theme.spacing(-0.25)}`,
```

Comment:
No need to wrap it in string here

Reply:
Typescript was complaining on a bunch of them, this was just my workaround. I will give this another try!

Reply:
My TS compiler seems to have done odd stuff throwing these errors. After a restart of vs code, it went fine - I corrected it here, and more is coming up in the spacings-refactoring PR.